superset/views/geocoding.py

```diff
@@ -212,7 +205,7 @@ def geocode(self) -> Response:
                 f"Failed to add columns/ data after geocoding {e.orig}"
```

Comment:
shouldn't this be "failed to add data..." only as the columns are created before geocoding? (Specifically mentioned in the documentation)